README.md

```diff
@@ -1,7 +1,21 @@
-# Open Application Platform
+# <img src="oap_logo.png" width="165" height="90"> Open Application Platform
```

Comment:
Could you approve the replacing image with OAP maintainer, please
@vovakyrychenko
Or we can vote fsomewhere or the best ico :)